src/controllers/identity/complete.ts

```diff
@@ -2,8 +2,26 @@ import { Request, Response } from "express";
 
 export function completeSavedGet(req: Request, res: Response) {
   res.render("identity/complete/saved");
+  if (req.session) {
```

Comment:
surprised ts doesn't make you check for the existence of journey too 🤔 

Would have thought we needed:

```
if (req.session && req.session.journey) {
``

Reply:
`req.session` is an implicit `any` type so as long as the `req.session` object isn't `undefined` then all bets are off